core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -78,18 +75,24 @@
 
   /** If the stream is closed, this can only go from false to true. */
   protected boolean mClosed;
-  /** Whether or not the current block should be cached. */
-  protected boolean mShouldCacheCurrentBlock;
-  /** Include incomplete blocks if Alluxio is configured to store blocks in Alluxio storage. */
-  private final boolean mShouldCachePartiallyReadBlock;
- /** Current position of the stream. */
+  /**
+   * Current position of the file instream. This is not always sync-ed with block instream or
```

Comment:
`not always sync-ed` -> `may be out of sync` 
Q: why they could be out of sync?